hardhat.config.ts

```diff
@@ -63,6 +62,7 @@ const chainIds = {
     arbitrum: 42161,
     "arbitrum-goerli": 421613,
     avalanche: 43114,
+    "avalanche-fuji": 43113,
```

Comment:
This might be too nitty, but maybe we could sort the `chainIds` in alphabetical order? We could also sort the `case` statements in `getChainConfig` and `explorereUrl` and the network names in `HardhatUserConfig`.

Reply:
Yep, much better this way!